kernel/will-it-scale.py.data/will-it-scale.yaml

```diff
@@ -0,0 +1,129 @@
+# To run all tests use following stanza instead
+#testcase: !mux
+#    All:
+#        name: All
+#        postprocess: True
```

Comment:
I feel these comment statement can be removed as user already knowing how to use ? May this can move as one of combination

Reply:
Yes, have removed it from yaml and added it to README.